setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 dependencies = [
-    "multidict==5.1.0",  # Avoid 5.2.0 due to Avast
+    "multidict==6.0.2",  # Avoid 5.2.0 due to Avast
```

Comment:
We don't use this directly so we can just drop it.  `aiohttp` will bring it in for us.

```suggestion
```